templates/server/config/_autoreload.js

```diff
@@ -0,0 +1,18 @@
+var path = require('path');
+
+module.exports.autoreload = {
+  active: true,
```

Comment:
Is that something to check in the settings though, or should I maybe make a PR upstream?